src/common/test-json.c

```diff
@@ -117,6 +117,31 @@ test_get_int (TestCase *tc,
 }
 
 static void
+test_get_double (TestCase *tc,
+                 gconstpointer data)
+{
+  gboolean ret;
+  gdouble value;
+
+  ret = cockpit_json_get_double (tc->root, "number", 0, &value);
+  g_assert (ret == TRUE);
+  g_assert_cmpint (value, ==, 55.4);
```

Comment:
`_cmpint()` for comparing floating point numbers really sounds wrong. Unfortunately `_cmpfloat()` wouldn't work either as you can't compare floats for equality reliably; I see no "equal up to given precision" comparison function on https://developer.gnome.org/glib/stable/glib-Testing.html#g-assert-cmpfloat, so I guess you need to implement this yourself? (Python's [assertAlmostEqual()](https://docs.python.org/2/library/unittest.html#unittest.TestCase.assertAlmostEqual) is perfect for that..)